### PR TITLE
DCOS-8392 Update Jobs struct to get last run status from history

### PR DESCRIPTION
**Pull request by pierlo-upitup**
The API introduced a change where the status property is removed and we
need to embed the history to grab the last run status.

screen shot 2016-07-28 at 14 40 26

Relevant tests are passing:

screen shot 2016-07-28 at 14 40 17

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @philipnrmn

**Comment by orlandohohmeier** [comment]
LGTM 💯 ✨

**Comment by pierlo-upitup** [comment]
@orlandohohmeier last commits add support for the tooltip

tooltip

cc @leemunroe

**Comment by leemunroe** [comment]
Yay!

File: src/js/pages/jobs/JobsTable.js [hunk 1]
```diff
@@ -68,12 +68,19 @@ class JobsTable extends React.Component {
...
     return this.props.jobs.map(function (job) {
       let isGroup = job instanceof Tree;
-      let lastRunStatus = null;
+      let lastRun = null;
       let schedules = null;
       let status = null;
 
       if (!isGroup) {
-        lastRunStatus = job.getLastRunStatus().status;
+        let lastRunsSummary= job.getLastRunsSummary();
```
Comment (philipnrmn) [inline src/js/pages/jobs/JobsTable.js]: missing space 😱

**Comment by philipnrmn** [comment]
This looks good! It's now missing the `LAST RUN` title in the header though:

screen shot 2016-07-28 at 6 03 21 pm

**Comment by philipnrmn** [comment]
I think you need to update the `lastRunStatus` key in `JobTableHeaderLables.js`

File: src/js/pages/jobs/JobsTable.js (hunk 1 above)
Comment (pierlo-upitup) [inline src/js/pages/jobs/JobsTable.js]: Actually why don't we have a eslint rule for this?

**Comment by pierlo-upitup** [comment]
All comments addressed.

**Comment by orlandohohmeier** [comment]
Thanks for adding the tooltips! LGTM 🚢

**Comment by mlunoe** [comment]
Please make sure integration tests are run again after changes are made. This PR broke integration tests. The fix is implemented here: https://github.com/dcos/dcos-ui/pull/865

**Comment by philipnrmn** [comment]
(I think it might actually be #885)

Thanks for catching and fixing this, @mlunoe.

**Comment by mlunoe** [comment]
Oh! Right on

**Pull request merged**